### PR TITLE
HW_Lesson_7_First_try

### PR DESCRIPTION
**Pull request by rodion-groshev**

### CODE REVIEW

File: HW_Lesson_7/HW_L7_Task_1.py [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+"""
+Implement your own print function. It should work on all operating systems. You can't use build-in print function
+"""
+
+
+def print_func(*args):
+    return sys.stdout.write(f"{str(*args)}\n")
+
+
+print_func("Hello world!")
```
Comment (bohdandovbsyh) [inline HW_Lesson_7/HW_L7_Task_1.py]: please do not forget about if name == main

File: HW_Lesson_7/HW_L7_Task_2.py [hunk 2]
```diff
@@ -0,0 +1,14 @@
+"""
+Implement your realization of the function filter
+"""
+
+
+def function_filter(callback, sequence):
```
Comment (bohdandovbsyh) [inline HW_Lesson_7/HW_L7_Task_2.py]: please add annotations in all tasks

**Review by bohdandovbsyh — APPROVED** [review]

File: HW_Lesson_7/HW_L7_Task_1.py (hunk 1 above)
Comment (rodion-groshev) [reply HW_Lesson_7/HW_L7_Task_1.py]: Ok, done

File: HW_Lesson_7/HW_L7_Task_2.py (hunk 2 above)
Comment (rodion-groshev) [reply HW_Lesson_7/HW_L7_Task_2.py]: Done

**Pull request merged**